### DISCUSSION TITLE
Tips on creating new models from datasets

### DISCUSSION THREAD

**Discussion by tospe** [nasa/prog_models#254] [Q&A] [answered]
Hi there,

Do you have any tips on creating models from datasets?

I saw #248 I believe once is completed can help.

Thanks in advance

**Comment by teubert** [comment, marked as answer]
Great question, 

We hope to explore data-driven models in the future (e.g., RNN). I hope to include tools for that in future releases. 

Without data-driven models in this package, the best approach is to try to represent the behavior using the physical characteristics (e.g., like done in the Battery models included). And use the data to validate your understanding.

Note: v1.3 release included the datasets feature, which helps provide access to data. See examples.datasets for more details

If this is something you would like to work with us on- let us know. We're always looking for partners.

**Reply by teubert** [reply]
@matteocorbetta: Do you have anything to add here?
